src/tools/pontoon/teams_projects.md

```diff
@@ -123,16 +135,15 @@ The page header contains useful information about the current project (same as t
 
 ![Header of localization page](../../assets/images/pontoon/teams_projects/localization_page_header.png)
 
-There are 6 tabs available:
+Underneath the header, the following tabs are available to navigate between various Team page panels:
+
 * **Resources**: files available in the project.
 * **Tags**: a list of tags defined for this project. This tab will be visible only in projects where tags are enabled. For more information about tags, see [this page](translation_workspace.md#tags).
 * **Contributors**: a list of active contributors with their statistics, filterable by time. Unlike the tab available in the Team page, this only includes contributions to localization of the project for the current locale.
-* [**Insights**](#insights-graphs): contains data and trends presented in a graphical format about review activity and translation activity.
-* **Project info**: information about the project.
+* [**Insights**](#insights-graphs): data and trends on contributor and localization activity.
+* **Project info**: same content as the Info tab in the Project page.
 * **Team info**: same content as the Info tab in the Team page.
 
-In addition to the Localization page for each locale/project pair, there is also a special Localization page that allows access to all resources for all projects enabled for a specific locale. This can be accessed from the `/{LOCALE}/all-projects/all-resources` URL (e.g. [pontoon.mozilla.org/it/all-projects/all-resources](https://pontoon.mozilla.org/it/all-projects/all-resources) for Italian).
-
```

Comment:
This is not a Localization page.

Reply:
`In addition to the Localization page for each locale/project pair, there is also a special page that allows access to all resources for all projects enabled for a specific locale. This can be accessed from the `/{LOCALE}/all-projects/all-resources` URL (e.g. [pontoon.mozilla.org/it/all-projects/all-resources](https://pontoon.mozilla.org/it/all-projects/all-resources) for Italian).`

Perhaps we can just delete "Localization" then instead of removing the whole thing.

Reply:
(@mathjazz - tagging just because this isn't showing up in the main PR view)

Reply:
To me this paragraph is more suitable for the [Translation workspace](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/translation_workspace.html#main-toolbar) doc, because the link leads there. I don't see the connection with the Localization dashboard.

Reply:
Alright, I've reworked it, ignoring using the URL and just adding it to the existing description of the teams page header.